simulariumio/cellpack/cellpack_data.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from enum import Enum
+import logging
+from typing import Any, Dict, List
+
+from ..data_objects import MetaData, UnitData, DisplayData, InputFileData
+from ..constants import DISPLAY_TYPE
+
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
+
+class HAND_TYPE(Enum):
+    RIGHT = "RIGHT"
+    LEFT = "LEFT"
+
+
+class CellpackData:
+    results_file: InputFileData
+    recipe_file_path: str
+    meta_data: MetaData
+    display_data: Dict[str, DisplayData]
+    geometry_type: DISPLAY_TYPE
+    time_units: UnitData
+    spatial_units: UnitData
+    plots: List[Dict[str, Any]]
+    handedness: HAND_TYPE
+    geometry_url: str
+
+    def __init__(
+        self,
+        results_file: InputFileData,
+        recipe_file_path: str,
+        meta_data: MetaData = None,
+        display_data: Dict[str, DisplayData] = None,
+        geometry_type: DISPLAY_TYPE = DISPLAY_TYPE.PDB,
+        time_units: UnitData = None,
+        spatial_units: UnitData = None,
+        plots: List[Dict[str, Any]] = None,
+        handedness: HAND_TYPE = HAND_TYPE.RIGHT,
+        geometry_url: str = "https://raw.githubusercontent.com/mesoscope/cellPACK_data/master/cellPACK_database_1.1.0/geometries/",  # noqa: E501
+    ):
+        """
+        This object holds simulation trajectory outputs
+        from Cellpack (http://www.cellpack.org) and plot data
+
+        Parameters
+        ----------
+        results_file: InputFileData
+            A InputFileData object containing the string path
+            or string contents of the Cellpack results output txt file.
+        recipe_file_path: str
+            A containing the string path
+            or of a Cellpack recipe that was used to
+            produce the results_file. The recipe name must match in both
+            files for the converter to run.
+        meta_data: MetaData (optional)
+            An object containing metadata for the trajectory
+            including box size, scale factor, and camera defaults
+            NOTE: When passing in a scale_factor use the number relative to a normal
+            cellPACK recipe, it will be scaled an additional 10% because of the
+            conversion from cellPACK to Simuarlarium.
+        display_data: Dict[str, DisplayData] (optional)
```

Comment:
This `display_data` doesn't seem to be used? use it to override if provided?

Reply:
I updated it to use the display data